konfera/models/organizer.py

```diff
@@ -0,0 +1,24 @@
+from konfera.models.countries import COUNTRIES
+from django.db import models
+from konfera.models.abstract import KonferaModel
+
+
+class Organizer(KonferaModel):
```

Comment:
I think @ricco386 meant that only the `title` should be required and the other fields should have the `blank=True` attribute.